releasenotes/notes/get_qpy_version-75791c063af4e851.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade_qpy:
```

Comment:
This is a new feature, upgrade notes are typically used for things that require users to take manual action when upgrade to the new release.

```suggestion
features_qpy:
```